pkgs/shells/mrsh/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkgconfig, readline }:
+
+stdenv.mkDerivation rec {
+  pname   = "mrsh";
+  version = "ef21854fc9ce172fb1f7f580b19a89d030d67c65";
+
+  src = fetchFromGitHub {
+    owner = "emersion";
+    repo = "mrsh";
+    rev = version;
+    sha256 = "1iyxmwl61p2x9v9b22416n4lnrlwjqyxybq35x8bcbjxkwypp943";
+  };
+
+  nativeBuildInputs = [ meson ninja pkgconfig ];
+  buildInputs = [ readline ];
+
+  meta = with stdenv.lib; {
+    description = "A minimal POSIX shell";
+    homepage = https://mrsh.sh;
```

Quoting URLs is preferred now.

```suggestion
    homepage = "https://mrsh.sh";
```